src/components/filters/FilterBy/PriceFilter.jsx

```diff
@@ -9,15 +12,15 @@ export default function PriceFilter() {
             type="range"
             min="1000"
```

Comment:
I see that you have products below 1000 too. So, u may wanna start range from a lower number.

Reply:
Made the changes, thanks!